indico/web/assets/bundles.py

```diff
@@ -172,7 +172,8 @@ def rjs_bundle(name, *files):
                'palettepicker.js',
                'itempicker.js',
                'sortablelist.js',
-               'categorynavigator.js'))
+               'categorynavigator.js',
+               '../modules/eventcreation.js'))
```

Comment:
Is this intended? Why not adding it as a standalone bundle within `module_js` like we do for all the other JS files under `js/indico/modules/`?